packages/bvaughn-architecture-demo/components/sources/PointPanelTimeline.module.css

```diff
@@ -70,18 +74,18 @@
 
 .HitPointTimelineBackground {
   width: 100%;
-  height: 0.5em;
-  background-color: var(--point-panel-timeline-background-color);
-  border-radius: 0.5em;
+  height: 0.35em;
```

Comment:
I've been trying to stick with increments of `0.25em` so we (a) avoid fractional sizes and (b) have a consistent grid for things to line up to. Do we really need 0.35 and 0.8 in this file?

Reply:
We don't. I can probably take this down to 0.25, but if 0.35 looks better we can add height so that it is 0.5 as a unit.